src/ext-test/java/org/opentripplanner/ext/fares/impl/DefaultFareServiceTest.java

```diff
@@ -50,8 +58,10 @@ void simpleZoneBasedFare() {
 
     var legProducts = fare.getLegProducts().get(itin.getTransitLeg(0));
     assertEquals(
-      "[FareProductUse[id=1d270201-412b-3b86-80f6-92ab144fa2e5, product=FareProduct{id: 'F:airport-to-city-center', name: 'regular', amount: $10.00}]]",
-      legProducts.toString()
+      List.of(
+        new FareProductUse("1d270201-412b-3b86-80f6-92ab144fa2e5", AIRPORT_TO_CITY_CENTER_PRODUCT)
```

Comment:
Do we need a comment reminding how the product use ids are computed? (They seem kinda magic if you don't dive into `FareProduct.uniqueInstanceId(startTime)`).